easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Binary'
+
+name = 'orca'
```

Comment:
This name for the package will be very confusing with `ORCA`. I would use the name `plotly-orca` which is the same name used by the package in conda.

Reply:
Perhaps, but we already have an older version in the repo.

Reply:
Yes, I just updated what was already there. Happy to proceed with the consensus view.

Reply:
my vote would be for renaming both the existing easyconfig and this one, as well as updating NanoComp-1.10.1-intel-2019b-Python-3.7.4.eb which depends on it. `plotly-orca` seems appropriate.

Reply:
Done in #12015